packages/customWidgets/image-viewer-web/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "image-viewer-web",
   "widgetName": "ImageViewer",
-  "version": "1.3.3",
+  "version": "1.3.2",
```

Comment:
Are we supposed to go down in version? 😄

Reply:
Yea, noted it in the commit message. Latest release on the Marketplace is v1.3.1.